### ISSUE TITLE
Wired VR headsets are not working using firefox browser

### ISSUE THREAD

**Issue by LExit89** [mozilla/hubs#5377]
**Description**
no accessibility with wired VR headset (HTC VIVE / VALVE INDEX etc) to any hubs scene using firefox browser, access window is missing

**To Reproduce**
1. Go to 'a mozilla hubs scene'
2. Click on 'enter in VR / in VR betreten'
4. See that there is only the "hub.link" window to access with wireless headset

**Expected behavior**
I want to enter the scenes with wired headsets and using firefox, the window to access with wired VR is missing

**Screenshots**

**Hardware** 
 - Device: HTC VIVE pro 2 / VALVE INDEX
 - OS: Windows 10
 - Browser: Firefox
<img width="920" alt="vrheadset" src="https://user-images.githubusercontent.com/93397719/164664787-48f47669-9681-4510-9a4b-98046b17cf16.PNG">

**Comment by matthewbcool** [comment]
Firefox does not enable webXR by default. Can be enabled behind the `dom.vr.webxr.enabled` flag. We have this documented in our discord thread but adding to here for others as well.

Does this issue persist with webXR enabled?

**Comment by michela** [comment]
More test feedback. Unclear what Virtual Desktop purpose for testing this (previously launched PC VR Hubs outside of VR) is but have recorded result here as well.

SteamVR build 964 5429
Valve Index HMD
SteamVR2 lighthouses
Chrome for Windows 106.0.5249.119 (Official Build) (64-bit) (cohort: 106_stable_119) 
Microsoft Edge 106.0.1370.42 (Official build) (64-bit) 

From Virtual Desktop

Start Virtual Desktop build 9570767
Open Hubs URL in CHrome
Enter from Device
Virtual Desktop closes
Browser opens - on Steam home space popup  "<browser 160> waiting. Next" remains onscreen, can hear room entry audio in HMD but room never opens
* Chrome
* Edge

From desktop

Open Hubs URL in Chrome
Enter from Device
Room opens on desktop and is functional
HMD remains in home space

Other test

https://immersive-web.github.io/webxr-samples 

VR support detected

https://immersive-web.github.io/webxr-samples/immersive-vr-session.html

Enter VR 

"<browser 106> Waiting" again

Suggesting this is not Hubs issue.
